core/services/relay/evm/size_helper.go

```diff
@@ -0,0 +1,72 @@
+package evm
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/abi"
+
+	relaytypes "github.com/smartcontractkit/chainlink-relay/pkg/types"
+)
+
+func GetMaxSizeFormEntry(n int, entry *CodecEntry) (int, error) {
+	if entry == nil {
+		return 0, relaytypes.InvalidTypeError{}
+	}
+	return GetMaxSize(n, entry.Args)
+}
```

Comment:
What about a method? Can shorten the name and drop the nil check:
```suggestion
func (e *CodecEntry) GetMaxSize(n int) (int, error) {
	return GetMaxSize(n, e.Args)
}
```

Reply:
sg, I'll move it into the right file.

Reply:
Ah sorry, I must have grabbed an older version  I checked it out again after merge conflicts :(.  Will fix again.